### ISSUE TITLE
Windows Proxy does not turn off after disabling shadowsocks

### ISSUE THREAD

**Issue by someoneIG** [shadowsocks/shadowsocks-windows#3227]
### Describe the bug
Windows Proxy does not turn off after disabling shadowsocks

### Environment
- Shadowsocks client version: 4.4.0.0 and 4.1.8.0
- OS version: windows 10  21H1
- .NET version:  4.8.04084

### What did you expect to see?
To disable the Windows proxy

### What did you see instead?
windows proxy still is ON after disabling shadowsocks and I have to disable it manually

```
https://pasteboard.co/KgTEO8on.png
```

**Comment by chenshaoju** [comment]
1.Exit the Shadowsocks.
2.Disable Proxy manually if the proxy is still on.
3.Go to ss_win_temp folder at Shadowsocks.exe folder.
4. There is a file name `user-wininet.json`, delete it.
5. start Shadowsocks again.

**Comment by someoneIG** [comment]
unfortunately didn't help

**Comment by chenshaoju** [comment]
Odd, If you do not start Shadowsocks, and reboot the computer, The proxy settings will auto enable or not?

**Comment by someoneIG** [comment]
no
only if the proxy settings are on, these settings will be on even after reboot

**Comment by hasanizadt** [comment]
how can Disable Proxy manually if the proxy is still on ?

**Comment by chenshaoju** [comment]
Right-click Shadowsocks in the windows taskbar tray icon area, Select `System Proxy`, and click `Disable`.
